source/core/slang-platform.cpp

```diff
@@ -105,8 +125,8 @@ SLANG_COMPILE_TIME_ASSERT(E_OUTOFMEMORY == SLANG_E_OUT_OF_MEMORY);
 
 /* static */void SharedLibrary::appendPlatformFileName(const UnownedStringSlice& name, StringBuilder& dst)
```

Comment:
Not related to this PR, but should this be `appendPlatformFileExtension`?

Reply:
Unfortunately no - because on some platforms a shared library name isn't just an extension. On linux it's prefixed with lib for example.

Reply:
Ah, right. Then `append` feels a bit misleading to. This is almost more `calcPlatformFileName`.